src/SConscript

```diff
@@ -47,6 +47,8 @@ def scons():
     # Generic DAOS includes
     env.AppendUnique(CPPPATH=[Dir('include').srcnode()])
     env.AppendUnique(CPPPATH=[Dir('include')])
+    il_env.AppendUnique(CPPPATH=[Dir('include').srcnode()])
```

Comment:
I think Dir('include') will add both paths to command line

Reply:
I had some trouble with this PR, once it's working I will try selectively backing parts out to check things this.

Reply:
It needs both.  The srcnode to pick up the source, and the one without to pick up the daos_version.h